Adding Peterson Algorithm

RankJay opened the pull request:
Adding a most simplified version of the Peterson algorithm according to my understanding

## Tasks Done 
-  Added EntrySection, CriticalSection, and ExitSection function as a part of the algorithm.
-  Had two process to carry out the implementation

## Tasks Not Done
- To add a sleep function that depicts the process executing under critical section
- To add threads (Async function in Javascript) to automate the working of algorithm dynamically.

RankJay committed:
Adding Peterson Algorithm

Adding a most simplified version of the Peterson algorithm according to my understand

RankJay committed:
Updated with dynamically executing instance

RankJay commented:
Updated with a basic dynamically executing instance of the algorithm

## Tasks Done 
-  Added EntrySection, CriticalSection, and ExitSection function as a part of the algorithm.
-  Had two process to carry out the implementation
- To add a sleep function that depicts the process executing under critical section

## Tasks Not Done

- To add threads (Async function in Javascript) to automate the working of the algorithm dynamically. 

#### ```Half Done as to async won't work here, looking for a substitute, else we can use this```

RankJay committed:
Add files via upload

RankJay committed:
Add files via upload

RankJay committed:
Added working HTML which can use CSS animation

RankJay committed:
Added CSS with interactive animation

krish-bhanushali commented:
Dear @RankJay it is really good. Once everyone completes the work by thursday we will create an introductory page and then will merge.

RankJay committed:
Updated with CSS transitions

RankJay commented:
Updated with a basic dynamically executing instance of the algorithm

## Tasks Done 
-  Added EntrySection, CriticalSection, and ExitSection function as a part of the algorithm.
-  Had two process to carry out the implementation
- To add a sleep function that depicts the process executing under critical section

## Tasks Not Done
- CSS Transitions have been added though they do not work up to mark, and need some specific time frames as well as locations to work perfectly
- To add threads (Async function in Javascript) to automate the working of the algorithm dynamically. 

#### ```Half Done as to async won't work here, looking for a substitute, else we can use this```

RankJay committed:
Added navBar

RankJay committed:
Added navBar

RankJay committed:
Rename Peterson.js to Peterson/Peterson.js

RankJay committed:
Added final Peterson.css

RankJay committed:
Added final Peterson.html

RankJay committed:
Added final Peterson.js

RankJay committed:
Added interest n turn visualization

RankJay committed:
Added interest n turn visualization

RankJay committed:
Added interest n turn visualization
